config/routes.php

```diff
@@ -32,6 +32,9 @@
     Route::get('/')
         ->action([SiteController::class, 'index'])
         ->name('site/index'),
+    Route::get('/locale')
```

Comment:
Using GET to change application state isn't a very good idea.

Reply:
It's just for shortcut purpose while developing.